drivers/gles2/rasterizer_storage_gles2.h

```diff
@@ -1182,10 +1182,13 @@ class RasterizerStorageGLES2 : public RasterizerStorage {
 		struct External {
```

Comment:
Unrelated to this PR, but the use of the term `External` might introduce confusion with `GL_TEXTURE_EXTERNAL_OES` textures.
Might be worth removing the confusing by renaming the struct in another PR.

Reply:
Well I got there first ! :P

J/k but it is interesting, I never really got why that extension is called TEXTURE_EXTERNAL, from what I understand it simply allows the texture to be shared amongst processes so one process could be writing say camera data into it (ARCore) while another process uses it to render. What the underlying magic is that makes it special I'm not sure.

But the external texture feature in ARVR is a different thing and was added a while back, here we refer to a render texture supplied externally by a plugin. I'm all for a better name but we should apply it across the board, not just this struct but also the method call in the ARVR plugin should be renamed (seeing we use a struct of function pointers changing the name of a pointer won't have that much impact). I suggest though that we do this in Godot 4, I've been wanting to rename all the ARVR objects to XR, might aswell do a bunch of improvements in the naming of things.